extensions/amp-fit-text/0.2/storybook/Basic.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '../../../../src/preact';
+import {FitText} from '../fit-text';
+import {number, text, withKnobs} from '@storybook/addon-knobs';
+import {withA11y} from '@storybook/addon-a11y';
+
+export default {
+  title: 'FitText',
+  component: FitText,
+  decorators: [withA11y, withKnobs],
+};
+
+const fitTextStyle = {
+  border: '1px solid black',
+  display: 'block',
+  width: 300,
+  height: 200,
+};
+
+export const _default = () => {
+  const minFontSize = number('minFontSize', 6);
+  const maxFontSize = number('maxFontSize', 72);
+  return (
+    <FitText
+      minFontSize={minFontSize}
+      maxFontSize={maxFontSize}
+      width="300"
```

Comment:
Yes. Looking through this, I'd definitely would rather like to see this passed in `style`.

Reply:
You mean ignore `width`/`height` props entirely, reorder priority so `style.width`/`style.height` takes precedence, or keep as is but only update the Storybook samples? Current the FitText component prefers props -> `style` props -> default `100%`

Reply:
Yes. Storybook can supply size to the component anyway it likes. E.g. via `style` prop. Or implicitly by making `amp-fit-text` a `position: absolute; inset: 0` inside some container, etc. I think `style` is definitely fine. But I wanted to see if we can make `width`/`height` completely unobservable to us.